ui/app/services/path-help.js

```diff
@@ -0,0 +1,89 @@
+// Low level service that allows users to input paths to make requests to vault
+// this service provides the UI synecdote to the cli commands read, write, delete, and list
+import Service from '@ember/service';
+
+import { getOwner } from '@ember/application';
+import { expandOpenApiProps, combineAttributes } from 'vault/utils/openapi-to-attrs';
+import { resolve } from 'rsvp';
+
+export function sanitizePath(path) {
+  //remove whitespace + remove trailing and leading slashes
+  return path.trim().replace(/^\/+|\/+$/g, '');
+}
+
+export default Service.extend({
+  attrs: null,
+  ajax(url, options = {}) {
+    let appAdapter = getOwner(this).lookup(`adapter:application`);
+    let { data } = options;
+    return appAdapter.ajax(url, 'GET', {
+      data,
+    });
+  },
+
+  getNewModel(modelType, backend, owner) {
```

Comment:
I think commenting this method (what it does) as well as the various lines would be super helpful